tests/setup.go

```diff
@@ -25,13 +25,13 @@ var (
 
 	dymensionImage = ibc.DockerImage{
 		Repository: DymensionMainRepo,
-		Version:    dymensionVersion,
+		Version:    "55d36971",
 		UidGid:     "1025:1025",
 	}
 
 	rollappImage = ibc.DockerImage{
-		Repository: RollappMainRepo,
-		Version:    rollappVersion,
+		Repository: "ghcr.io/decentrio/rollapp-evm",
+		Version:    "e2e-amd",
```

Comment:
we should be using latest rollapp-evm image build from dymension. For now, the proposed solution is fine to use hardcoded image stored on decentrio. We should create a separate PR on rollapp-evm repo, though and update this code to use that image

Reply:
Got it! My plan is the same, I will do it when the tests pass.